packages/perseus/src/widgets/radio/radio.new.tsx

```diff
@@ -29,6 +29,7 @@ export interface RenderProps {
     deselectEnabled?: boolean;
     choices: ReadonlyArray<RadioChoiceWithMetadata>;
     selectedChoices: ReadonlyArray<PerseusRadioChoice["correct"]>;
+    showSolutions?: ShowSolutions;
```

Comment:
Huh strange, I wonder why this was removed as I don't see anything calling it here being added, and we're not getting any linter errors. Either way, it makes sense for this line to stay to me as we're definitely using showSolutions further down the radio hierarchy